embeddable-say-editor/app/components/simple-editor.js

```diff
@@ -108,6 +105,12 @@ import {
   defaultTableOfContentsPluginConfig,
   mergeConfigs,
 } from '../config/defaults';
+import {
+  bulletListWithConfig,
+  listItemWithConfig,
+  orderedListWithConfig,
+} from '@lblod/ember-rdfa-editor/plugins/list';
+import { headingWithConfig } from '@lblod/ember-rdfa-editor/plugins/heading';
```

Comment:
I hate to be pedantic, but what dev tool is it that is constantly mangling the imports in JS files? (it seems to be only in those, not in TS files). It seems like all of these files have a bunch of random newlines for no reason. It would make sense for the `./...` import to be last, possibly after a newline, but there's no reason to have these there.

Maybe this only bothers me though and I'm not sure why, it doesn't really change anything...

Reply:
I also like `../`  at the end of import. There's an ESLint plugin for that too :D 
We can introduce it 🤷🏻‍♂️